src/pages/EnablePayments/TermsStep.js

```diff
@@ -0,0 +1,164 @@
+import PropTypes from 'prop-types';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Navigation from '../../libs/Navigation/Navigation';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import Checkbox from '../../components/Checkbox';
+import styles from '../../styles/styles';
+import Button from '../../components/Button';
+import {activateWallet} from '../../libs/actions/BankAccounts';
+import CONST from '../../CONST';
+import TextLink from '../../components/TextLink';
+import compose from '../../libs/compose';
+import ONYXKEYS from '../../ONYXKEYS';
+
+const propTypes = {
+    walletTerms: PropTypes.shape({
+        loading: PropTypes.bool,
+    }),
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    walletTerms: {
+        loading: false,
+    },
+};
+
+class TermsStep extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.toggleDisclosure = this.toggleDisclosure.bind(this);
+        this.togglePrivacyPolicy = this.togglePrivacyPolicy.bind(this);
+        this.state = {
+            hasAcceptedDisclosure: false,
+            hasAcceptedPrivacyPolicyAndWalletAgreement: false,
+            error: false,
+        };
+    }
+
+    toggleDisclosure() {
+        this.setState(prevState => ({hasAcceptedDisclosure: !prevState.hasAcceptedDisclosure}));
+    }
+
+    togglePrivacyPolicy() {
+        this.setState(prevState => ({hasAcceptedPrivacyPolicyAndWalletAgreement: !prevState.hasAcceptedPrivacyPolicyAndWalletAgreement}));
+    }
+
+    render() {
+        return (
+            <ScreenWrapper>
+                <HeaderWithCloseButton
+                    title={this.props.translate('termsStep.headerTitle')}
+                    onCloseButtonPress={() => Navigation.dismissModal()}
+                />
+                <View style={[styles.mh5, styles.flex1]}>
+                    <ScrollView>
+                        <Text>
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                        </Text>
+                    </ScrollView>
+                    <View style={[styles.flexRow, styles.mb4]}>
+                        <Checkbox
+                            isChecked={this.state.hasAcceptedDisclosure}
+                            onClick={this.toggleDisclosure}
+                        />
+                        <TouchableOpacity
+                            onPress={this.toggleDisclosure}
+                            style={[
```

Comment:
I think we should DRY this up ... this is a lot of composed styles and they're repeated below.

Reply:
Yeah, I agree.